digdag-client/src/main/java/io/digdag/client/api/RestSessionAttempt.java

```diff
@@ -48,6 +48,19 @@ default int getIndex()
 
     Optional<Instant> getFinishedAt();
 
+    default String getStatus() {
+        if (this.getCancelRequested()) {
+            return "KILLED";
+        }
+        if (this.getSuccess()) {
+            return "SUCCESS";
```

Comment:
If you don't have a strong opinion for using an uppercase string as a return value, why don't you return a lowercase string to keep consistency with the existing behaviour?